internal/net/grpc/logger/logger_test.go

```diff
@@ -29,41 +32,50 @@ func TestInit(t *testing.T) {
 		name       string
 		want       want
 		checkFunc  func(want) error
-		beforeFunc func()
+		beforeFunc func(*testing.T)
 		afterFunc  func()
 	}
 	defaultCheckFunc := func(w want) error {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "set logger success with verbosity level is not set",
+			checkFunc: func(w want) error {
+				if grpclog.V(1) {
+					return errors.New("verbosity level is set")
+				}
+				return nil
+			},
+		},
+		// this test case cannot be run because the once object is initialized
```

Comment:
I think you should clear sync once object with before and after func

Reply:
fixed